karton/core/backend.py

```diff
@@ -551,3 +554,16 @@ def check_bucket_exists(self, bucket: str, create: bool = False) -> bool:
         if create:
             self.minio.make_bucket(bucket)
         return False
+
+
+    def inc_object_counter(self, bucket: str, object_uid: str) -> int:
```

Comment:
Maybe use a different function name, like `inc_refcount` or `inc_references` or `add_reference` etc. It's not immediately clear from the name what this function does.

Reply:
Also we don't use the values returned, maybe just return None (but tihs is irrelevant if we rewrite this to use pipelines)